ec2/sc-product-ec2-linux-jumpcloud-development.yaml

```diff
@@ -0,0 +1,59 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: Linux EC2 ServiceCatalog product with Jumpcloud integration.
+Parameters:
+  PortfolioProvider:
+    Type: String
+    Description: Owner and Distributor Name
+  LaunchConstraintARN:
+    Type: String
+    Description: ARN of the launch constraint role for EC2 products.
+  PortfolioId:
+    Type: String
+    Description: The ServiceCatalog portfolio this product will be attached to.
+  RepoRootURL:
+    Type: String
+    Description: Root url for the repo containing the product templates.
+  StackDatetime:
+    Type: String
+    Description: Last updated datetime.
+    Default: ""
+Resources:
+  scec2linuxproduct:
+    Type: AWS::ServiceCatalog::CloudFormationProduct
+    Properties:
+      Name: '[DEVELOPMENT] EC2 Linux with Jumpcloud Integration'
+      Description: This product builds one Linux EC2 instance, either
+        Amazon Linux or Ubuntu, with Jumpcloud integration. It also creates a
+        SSM patch baseline, maintenance window, and patch task to scan for
+        and install operating system updates on the EC2 instance.
+      Owner: !Ref 'PortfolioProvider'
+      Distributor: !Ref 'PortfolioProvider'
+      SupportDescription: Sage IT
+      SupportEmail: it@sagebase.org
+      AcceptLanguage: en
+      SupportUrl: https://app.slack.com/client/T092YT0LV/CEFQD0KU1
+      ProvisioningArtifactParameters:
+        - Description: baseline version
+          Info:
+            LoadTemplateFromURL: !Sub '${RepoRootURL}ec2/sc-ec2-linux-jumpcloud-v1.0.0.yaml'
+          Name: v.1.0.0
+        - Description: !Sub 'Some minor update. Last updated ${StackDatetime}.'
```

Comment:
This is what triggers the update -- this text will change with every sceptre launch if the sceptre-date-resolver is used. Fortunately, this forces the template to be re-fetched.